app/views/research_sessions/topic.html.erb

```diff
@@ -0,0 +1,19 @@
+<%= render "research_sessions/progress" %>
+<div class="main-content">
+  <%= render 'error_summary' %>
+  <h1 class="visually-hidden">Research Topic</h1>
+  <%= form_for @research_session, url: wizard_path do %>
+      <%= labelled_text_area_tag(
+            :topic,
+            'What is the research or participation session about?',
+            @research_session.topic,
+            label_options: {
+              hint: 'The topic is the project is …[what is the research about?]',
```

Comment:
Split into hint / placeholder. Remove space between "is" and "…" as well as brackets.